docs/canvas/canvas-share-workpad.asciidoc

```diff
@@ -2,47 +2,94 @@
 [[workpad-share-options]]
 == Share your workpad
 
-When you are ready to share your workpad, create a PDF, or export your workpad.
+When you've finished your workpad, you can share it outside of {kib}.
+
+[float]
+[[export-single-workpad]]
+=== Export workpads
+
+Create a JSON file of your workpad that you can export outside of {kib}.
+
+. From your workpad, click the *Share workpad* icon in the upper left corner.
+
+. Select *Download as JSON*.
++
+[role="screenshot"]
+image::images/canvas-export-workpad.png[Export single workpad]
+
+Want to export multiple workpads? Go to the *Canvas workpads* page, select the workpads you want to export, then click *Export*.
 
 [float]
 [[create-workpad-pdf]]
 === Create a PDF
 
-To view your workpad outside of Kibana, generate a PDF. 
+Create a PDF copy of your workpad that you can save and share outside of {kib}.
 
 . If you are using a Gold or Platinum license, enable reporting in your `config/kibana.yml` file.
 
 . From your workpad, click the *Share workpad* icon in the upper left corner, then select *PDF reports*.
 
-. Click *Generate PDF*. 
+. Click *Generate PDF*.
 +
 [role="screenshot"]
 image::images/canvas-generate-pdf.gif[Generate PDF]
 
 [float]
-[[export-workpad]]
-=== Export your workpad
+[[create-workpad-URL]]
+=== Create a POST URL
 
-To share your workpad with another author, export it as a JSON file.
+Create the POST URL that automatically generates PDF reports with Watcher or a script.
+
+For more information, refer to <<automating-report-generation, Automating report generation>>.
+
+. If you are using a Gold or Platinum license, enable reporting in your `config/kibana.yml` file.
+
+. From your workpad, click the *Share workpad* icon in the upper left corner, then select *PDF reports*.
+
+. Click *Copy POST URL*.
++
+[role="screenshot"]
+image::images/canvas-create-URL.gif[Create POST URL]
 
 [float]
-[[export-single-workpad]]
-==== Export a single workpad
+[[add-workpad-website]]
+=== Share the workpad on a website
 
-. From your workpad, click the *Share workpad* icon in the upper left corner.
+beta[] To show off your workpad outside of {kib}, share a static version of the workpad on an external website.
```

Comment:
I'd leave out "show off your workpad outside of {kib}.  All the sections on this page do that.

What's the value of sharing a static version?

Reply:
This needs wordsmithing but some potential value props:
- Embed your workpad in any website
- Configure it to rotate or autoplay pages